tests/by-util/test_relpath.rs

```diff
@@ -1 +1,151 @@
-// ToDO: add tests
+use crate::common::util::*;
+
+struct TestCase<'a> {
+    from: &'a str,
+    to: &'a str,
+    expected: &'a str,
+}
+
+const TESTS: [TestCase; 10] = [
+    TestCase {
+        from: "A/B/C",
+        to: "A",
+        expected: "../..",
```

Comment:
You could use https://doc.rust-lang.org/std/path/constant.MAIN_SEPARATOR.html as the separator

Reply:
While searching for a solution for this I found out about `MAIN_SEPARATOR`, that's also when I commented about it in your PR ;)
Ultimately I decided against it, since I didn't want to blow up the test cases.